receiver/awscontainerinsightreceiver/internal/k8sapiserver/prometheus_scraper.go

```diff
@@ -29,7 +29,8 @@ const (
 	caFile             = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
 	collectionInterval = 60 * time.Second
 	// needs to start with "containerInsightsKubeAPIServerScraper" for histogram deltas in the emf exporter
-	jobName = "containerInsightsKubeAPIServerScraper"
+	jobName                        = "containerInsightsKubeAPIServerScraper"
+	serviceAccountTokenDefaultPath = "/var/run/secrets/kubernetes.io/serviceaccount/token" // #nosec
```

Comment:
what is nosec

Reply:
its to prevent lint checks from complaining about these hardcoded paths in code.